libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/MapRouteLine.kt

```diff
@@ -685,8 +686,14 @@ internal class MapRouteLine(
          *
          * @return either the layer ID if found else a default layer ID
          */
-        fun getBelowLayer(layerId: String?, style: Style): String =
-            style.layers.firstOrNull { it.id == layerId }?.id ?: LocationComponentConstants.SHADOW_LAYER
+        fun getBelowLayer(layerId: String?, style: Style): String {
+            return when (layerId.isNullOrEmpty()) {
+                false -> style.layers.firstOrNull { it.id == layerId }?.id
+                true -> style.layers.reversed().filter { it !is SymbolLayer }
+                    .firstOrNull { it.id != RouteConstants.MAPBOX_LOCATION_ID }
+                    ?.id
+            } ?: LocationComponentConstants.SHADOW_LAYER
+        }
```

Comment:
Noting that this is the implementation we currently use in production https://github.com/mapbox/mapbox-navigation-android/blob/e232efd36700e4f5f3345bd76ad99cb28076bb1e/libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/route/MapRouteLine.java#L411-L423